Modules/Loadable/Segmentations/Widgets/qMRMLSegmentsTableView.h

```diff
@@ -166,6 +170,9 @@ public slots:
   /// Set the text used to filter the segments in the table
   /// \sa textFilter
   void setTextFilter(QString textFilter);
+  /// Set the settings key used to specify whether standard terminologies are used for name and color.
+  /// \sa terminologySelectorSettingsKey
```

Comment:
Describe what is the default and what happens if the key is set to empty string.